README.md

```
@@ -3,10 +1,28 @@
+# AIO2024 homework
+## Table of contents:
+- [Setup Project](#setup-project)
+- [Content](#content)
+    - [Module 1: Towards AI and Data Science](#module-1-towards-ai-and-data-science)
+    - [Module 2: Calculus and Linear Algegra, and theirs Applications to AI](#module-2-calculus-and-linear-algegra-and-theirs-applications-to-ai)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)